pvlib/tests/iotools/test_psm3.py

```diff
@@ -21,7 +23,20 @@
     'Temperature Units', 'Pressure Units', 'Wind Direction Units',
     'Wind Speed', 'Surface Albedo Units', 'Version']
 PVLIB_EMAIL = 'pvlib-admin@googlegroups.com'
-DEMO_KEY = 'DEMO_KEY'
+
+
+@pytest.fixture(scope="module")
+def DEMO_KEY():
```

Comment:
let's name this function `api_key` and also rename the variables below for clarity.

Reply:
I named it nrel_api_key since there was a conflict with the get_psm3() keyword of api_key.